### ISSUE TITLE
I get an error: require is not defined

### ISSUE THREAD

**Issue by tinybug** [nwjs/nw.js#3808]
when I open soundcloud.com, I get an error: require is not defined.
![error](https://cloud.githubusercontent.com/assets/3439875/9704257/f74e580e-54d1-11e5-9a33-162fba70944e.png)
##

**Comment by tshinnic** [comment]
I get this also with plain Chrome.  And that's an answer of sorts, that if plain Chrome complains on this site, you could expect NW.js to complain also.  Thus not a problem unique to NW.js.

FF/IE/Edge do not complain.  

But... the minimized source soundcloud.com is fetching [vendor-5a147-88eda9d.js](https://a-v2.sndcdn.com/assets/vendor-5a147-88eda9d.js) looks very strange to me.  It looks like another minimizer has been run to produce AMD output?  I am sure it is all wonderful, but at least once it looked like it goofed, sticking the '!' that was supposed to be in front of a 'function' instead into a block comment "/*! .... */function ..."

Anyway, closable as not specific to NW.js?

**Comment by NicholasJohn16** [comment]
I'm getting this same error. I recently updated from NW 0.12.1 to 0.12.3 and that's when this issue cropped up.

I went through my app an commented out any parts that use `require` (it's a small app) and loaded up my app and the error went away. Just to double check I loaded up soundcloud.com in nw.js and Chrome and didn't see this error so I think it is something specific to NW.js.

I'm current using the [winstate.js](https://github.com/nwjs/nw.js/wiki/Preserve-window-state-between-sessions) in my app. I uncommented it and reran the app. The window never opens and I get the above error in the console. If I comment out just the `var gui = require('nw.gui');` line the error changes to `gui is not defined` so the issue must be with `require`. 

I created a test-app just to see if I could recreate the bug in a fresh install, but had no luck. Winstate.js ran perfectly with no errors so the issue must be someone in the update. @tinybug, by chance, did you update recently? Does anyone have an ideas as what could be causing this? Did I need to clear our some cache or temp files somewhere after updating?

Any assistance would be appreciated.
